src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -228,8 +228,25 @@ export default defineComponent({
             {
               label: this.$t('Video.Open Channel in Invidious'),
               value: 'openInvidiousChannel'
+            },
+            {
+              type: 'divider'
             }
           )
+
+          const hiddenChannels = JSON.parse(this.$store.getters.getChannelsHidden)
+          const channelShouldBeHidden = hiddenChannels.some(c => c === this.channelName || c === this.channelId)
```

Comment:
AFAIK channelName is not a thing anymore

Reply:
Would we not need the channel name checked here to support anyone that has legacy blocks via channel name?

Reply:
i could be missing something but

If im looking at the conversations in https://github.com/FreeTubeApp/FreeTube/pull/4230 i think everything is based on channel ID now

Reply:
I was just thinking of what would happen to a user who has blocked a channel by name then updates freetube

Reply:
Ah ok, scrolling through the other PR it does look like we are fine just not respecting legacy blocks https://github.com/FreeTubeApp/FreeTube/pull/4230#issuecomment-1783769677

In that case I can remove the channel name check